src/game/GameProcMain.cpp

```diff
@@ -7416,6 +7419,17 @@ bool CGameProcMain::OnMouseRDBtnPress(POINT ptCur, POINT ptPrev) {
     return true;
 }
 
+// Rotates the player's camera 180 degrees from the current position.
+bool CGameProcMain::OnMouseMBtnPress(POINT ptCur, POINT ptPrev) {
+    if (s_pPlayer->IsAlive())
+	{
+        if (VP_THIRD_PERSON == s_pEng->ViewPoint()) 
+            s_pEng->CameraYawAdd(100);
+	}
+
+    return true;
+}
+
```

Comment:
Instead of implementing it without animation, I suggest doing it as per official behavior in `CGameProcMain::Tick` in order for it to be animated per frame:
```c++
  if ( this->m_fRotateValue >= 0.0f && !CGameProcedure::s_pUIMgr->m_bDoneSomething )
  {
    v40 = s_fSecPerFrm * _PI2;
    v41 = v40 + this->m_fRotateValue;
    this->m_fRotateValue = v41;
    if ( !(v43 | v44) )
    {
      v45 = v41 - s_PI;
      this->m_fRotateValue = -1.0;
      v40 = v40 - v45;
    }
    if ( v40 != 0.0f )
    {
      m_eState = CGameBase::s_pPlayer->m_eState;
      if ( m_eState != PSA_DYING && m_eState != PSA_DEATH )
      {
        bThirdPersonView = CGameProcedure::s_pEng->m_eViewPoint == VP_THIRD_PERSON;
        fYaw = v40 / s_fSecPerFrm;
        if ( bThirdPersonView )
        {
          CGameEng::CameraYawAdd(CGameProcedure::s_pEng, fYaw);
        }
        else if ( !CGameBase::s_pPlayer->m_iSkillStepMaybe )
        {
          CPlayerMySelf::RotAdd(CGameBase::s_pPlayer, fYaw);
        }
      }
    }
  }
```